waku/common/message_test.go

```diff
@@ -187,17 +187,24 @@ func TestMessageSeal(t *testing.T) {
 	target := 32.0
 	params.WorkTime = 4
 	params.PoW = target
-	env.Seal(params) // nolint: errcheck
+	err = env.Seal(params)
+	if err != nil {
+		t.Logf("failed to seal envelope: %s", err)
+	}
 
 	env.CalculatePoW(0)
 	pow := env.PoW()
 	if pow < target {
 		t.Fatalf("failed Wrap with seed %d: pow < target (%f vs. %f).", seed, pow, target)
 	}
 
+	// TODO is this meant to fail? The WorkTime is set to 1 but the PoW is 1000000000.0
```

Comment:
Does anyone know if this test is intended to fail? The `WorkTime` is set to 1 but the `PoW` is 1000000000.0 . I would presume a fail is expected.

If a fail is expected we should add a check for the fail and pass the test if the error is a "not enough WorkTime" error.

Reply:
I'd agree, seems like they are expecting a fail

Reply:
Yep.